src/mapml/control/AnnounceScale.js

```diff
@@ -0,0 +1,67 @@
+export var AnnounceScale = L.Control.extend({
+    options: {
+      position: 'bottomleft',
+      maxWidth: 100,
+      updateWhenIdle: true,
+    },
+  
+    onAdd: function (map) {
+      var container = L.DomUtil.create('div', 'accessible-scalebar');
```

Comment:
I think you might want to maintain a "private" `_container` property, that you could return with a custom AnnounceScale.getContainer() method.